db/schema.rb

```diff
@@ -38,7 +36,6 @@
     t.text "name"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.integer "category_id"
```

Comment:
we should never be touching the schema! Idk why exactly but everyone freaks out about it and I'm sure its for a good reason. If we want to get rid of that row, best thing to do is create a migration to remove the migration.